### DISCUSSION TITLE
yun是否支持一个网页多种语言

### DISCUSSION THREAD

**Discussion by emerisly** [YunYouJun/hexo-theme-starry#224] [Q&A 疑难杂症] [answered]
想实现一个网站多种语言（同一内容，多个不同语言的markdown）
请问这个“个性化语言包” 是支持多种语言的设置吗？有更加具体的如何设置的教程吗
感谢呜
![image](https://user-images.githubusercontent.com/44758321/147375641-038fe5c8-4096-4da3-992b-ebcb81b02313.png)

**Comment by YunYouJun** [comment, marked as answer]
这种语言包，通常只是针对某些特定关键词的多语言翻译，如果你希望整个页面都有对应的多语言翻译。

直接新建新的页面，如 `/xxx.md` 与  `/en/xxx.md` 的方式要更为合适，这也是目前许多中英文文档采用的方式。
